db/columns/exceptions.py

```diff
@@ -40,3 +42,7 @@ class ExclusionError(Exception):
 
 class ColumnMappingsNotFound(Exception):
     pass
+
+
+class InvalidStringTruncation(Exception):
```

Comment:
Is the custom exception necessary? Wouldn't it be better to just handle the underlying exception at the higher level?

Reply:
You are right the higher-level DB-API exception is clear enough.